packages/babel-plugin-relay/__tests__/__snapshots__/BabelPluginRelayMacro-test.js.snap

```diff
@@ -0,0 +1,12 @@
+// Jest Snapshot v1, https://goo.gl/fbAQLP
+
+exports[`BabelPluginRelayMacro works 1`] = `
+"
+'use strict';
+
```

Comment:
Unlike in `babel-plugin-relay`, the import is not currently retained. I'm not sure if that's acceptable? AFAIK the import is retained with `babel-plugin-relay` only for the "unexpected invocation at runtime" warning (someone correct me if I'm wrong).

Reply:
If the import needs to be retained (or replaced with something else) then your macros can get the Program node from the `state` that's passed in and you can unshift an import yourself :+1:

Reply:
The import is not needed in Modern (classic pulls some runtime helpers from it).